tf2/include/tf2/LinearMath/Transform.h

```diff
@@ -1,321 +1,297 @@
+  {
+    m_basis.setRotation(q);
+  }
 
   /**@brief Set this transformation to the identity */
-        TF2_PUBLIC
-	void setIdentity()
-	{
-		m_basis.setIdentity();
-		m_origin.setValue(tf2Scalar(0.0), tf2Scalar(0.0), tf2Scalar(0.0));
-	}
-
-  /**@brief Multiply this Transform by another(this = this * another) 
+  TF2_PUBLIC
+  void setIdentity()
+  {
+    m_basis.setIdentity();
+    m_origin.setValue(tf2Scalar(0.0), tf2Scalar(0.0), tf2Scalar(0.0));
+  }
+
+  /**@brief Multiply this Transform by another(this = this * another)
    * @param t The other transform */
-        TF2_PUBLIC
-	Transform& operator*=(const Transform& t) 
-	{
-		m_origin += m_basis * t.m_origin;
-		m_basis *= t.m_basis;
-		return *this;
-	}
+  TF2_PUBLIC
+  Transform & operator*=(const Transform & t)
+  {
+    m_origin += m_basis * t.m_origin;
+    m_basis *= t.m_basis;
+    return *this;
+  }
 
   /**@brief Return the inverse of this transform */
-        TF2_PUBLIC
-	Transform inverse() const
-	{ 
-		Matrix3x3 inv = m_basis.transpose();
-		return Transform(inv, inv * -m_origin);
-	}
+  TF2_PUBLIC
+  Transform inverse() const
+  {
+    Matrix3x3 inv = m_basis.transpose();
+    return Transform(inv, inv * -m_origin);
+  }
 
   /**@brief Return the inverse of this transform times the other transform
-   * @param t The other transform 
+   * @param t The other transform
    * return this.inverse() * the other */
-        TF2_PUBLIC
-	Transform inverseTimes(const Transform& t) const;  
+  TF2_PUBLIC
+  Transform inverseTimes(const Transform & t) const;
 
   /**@brief Return the product of this transform and the other */
-        TF2_PUBLIC
-	Transform operator*(const Transform& t) const;
+  TF2_PUBLIC
+  Transform operator*(const Transform & t) const;
 
   /**@brief Return an identity transform */
-        TF2_PUBLIC
-	static const Transform&	getIdentity()
-	{
-		static const Transform identityTransform(Matrix3x3::getIdentity());
-		return identityTransform;
-	}
-
-        TF2_PUBLIC
-	void	serialize(struct	TransformData& dataOut) const;
+  TF2_PUBLIC
+  static const Transform & getIdentity()
+  {
+    static const Transform identityTransform(Matrix3x3::getIdentity());
+    return identityTransform;
+  }
 
-        TF2_PUBLIC
-	void	serializeFloat(struct	TransformFloatData& dataOut) const;
+  TF2_PUBLIC
+  void    serialize(struct        TransformData & dataOut) const;
 
-        TF2_PUBLIC
-	void	deSerialize(const struct	TransformData& dataIn);
+  TF2_PUBLIC
+  void    serializeFloat(struct   TransformFloatData & dataOut) const;
 
-        TF2_PUBLIC
-	void	deSerializeDouble(const struct	TransformDoubleData& dataIn);
+  TF2_PUBLIC
+  void    deSerialize(const struct        TransformData & dataIn);
 
-        TF2_PUBLIC
-	void	deSerializeFloat(const struct	TransformFloatData& dataIn);
+  TF2_PUBLIC
+  void    deSerializeDouble(const struct  TransformDoubleData & dataIn);
 
+  TF2_PUBLIC
+  void    deSerializeFloat(const struct   TransformFloatData & dataIn);
```

Comment:
Can we just make the spacing consistent with the rest of the code?  That is, one space after the return type, and one space separating the argument qualifiers?